Pods/EPContactsPicker.swift

```diff
@@ -288,10 +388,14 @@ open class EPContactsPicker: UITableViewController, UISearchResultsUpdating, UIS
                     return selectedContact.contactId != $0.contactId
                 }
             }
-            else {
+            else if (self.multiSelectContactLimit == 0 || self.selectedContacts.count < Int(self.multiSelectContactLimit)) {
                 cell.accessoryType = UITableViewCellAccessoryType.checkmark
+                cell.tintColor = EPGlobalConstants.Colors.nxYellow
                 selectedContacts.append(selectedContact)
             }
+          let enabled = self.multiSelectContactLimit == 0
```

Comment:
I think that you don't need to compare multiSelectContactLimit to 0

Reply:
self.multiSelectContactLimit == 0 means no limit
I didn't want to change the general contact picker logic